vimrc

```diff
@@ -187,6 +187,16 @@ let g:go_highlight_trailing_whitespace_error = 0
 
 let g:completor_auto_trigger = 0
 
+
+function! Ctag()
+  let ruby = match(expand("%"), ".rb")
+  if ruby != -1
+    exec ':!ctags -R --languages=ruby --exclude=".git\|.svn\|log\|tmp\|db\|pkg" . $(bundle list --paths)'
```

Comment:
Can the exclude bit be pulled out into a variable?

Reply:
Thanks, please see updated commit.